### PR TITLE
Add 1.19.6 support

### PR DESCRIPTION
**Pull request by julek-wolfssl**

### CODE REVIEW

File: nginx-1.19.6-wolfssl.patch [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+ 
+ 
+diff -ur nginx-1.19.6/src/stream/ngx_stream_proxy_module.c nginx/src/stream/ngx_stream_proxy_module.c
+--- nginx-1.19.6/src/stream/ngx_stream_proxy_module.c	2020-12-15 15:41:39.000000000 +0100
++++ nginx/src/stream/ngx_stream_proxy_module.c	2021-01-13 15:12:10.226632715 +0100
+@@ -2165,6 +2165,8 @@
+             return NGX_ERROR;
+         }
+ 
++        ngx_ssl_set_verify_on(cf, pscf->ssl);
```
Comment (SparkiDev) [inline nginx-1.19.6-wolfssl.patch:268]: Should this be happening for OpenSSL as well?

File: nginx-1.19.6-wolfssl.patch [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+ #else
+             CRYPTO_add(&issuer->references, 1, CRYPTO_LOCK_X509);
+diff -ur nginx-1.19.6/src/http/modules/ngx_http_proxy_module.c nginx/src/http/modules/ngx_http_proxy_module.c
+--- nginx-1.19.6/src/http/modules/ngx_http_proxy_module.c	2020-12-15 15:41:39.000000000 +0100
++++ nginx/src/http/modules/ngx_http_proxy_module.c	2021-01-13 15:12:10.226632715 +0100
+@@ -4932,6 +4932,8 @@
+             return NGX_ERROR;
+         }
+ 
++        ngx_ssl_set_verify_on(cf, plcf->upstream.ssl);
```
Comment (SparkiDev) [inline nginx-1.19.6-wolfssl.patch:215]: Should this be happening for OpenSSL too?

**Review by SparkiDev — CHANGES_REQUESTED** [review]

File: nginx-1.19.6-wolfssl.patch (hunk 1 above)
Comment (julek-wolfssl) [reply nginx-1.19.6-wolfssl.patch:268]: I don't really test the patched version against OpenSSL. My reasoning is that if a user wants to use Nginx with OpenSSL then they don't need our patch. I can macro this out if you would like.

Comment (SparkiDev) [reply nginx-1.19.6-wolfssl.patch:268]: Yes. Macro it out.

**Review by SparkiDev — APPROVED** [review]

**Pull request merged**